index.html

```diff
@@ -238,7 +238,7 @@ <h2 class="ab-title new">Skills</h2>
 
     <!-- contact me section -->
     <section id="abt-contact">
```

Comment:
- [ ] as mentioned by the previous reviewer, the `contact me` section should use display flex, and not the title , I would suggest adding these CSS styling : 

```CSS
#abt-contact {
    display: flex;
    flex-direction: column;
}
```